src/typeDefs.ts

```diff
@@ -48,6 +48,7 @@ type ViewIdOptions =
 export type PickerConfiguration = {
   clientId: string
   developerKey: string
+  AppId?: string
```

Comment:
Why AppId with an capital A letter?

Reply:
Looks like that must have been a typo. appId is already included on line 60, so I'll update the PR to remove this line.